tests/util/ndhm.js

```diff
@@ -329,7 +347,98 @@ async function interceptNdhmDemographics() {
 
     await intercept(process.env.bahmniHost + "hiprovider/v0.5/hip/ndhm-demographics", response, 1)
 }
+async function interceptEmailPhoneInit() {
+    var response = {
+        method: 'POST',
+        status: 202
+    }
+
+    await intercept(process.env.bahmniHost + "/hiprovider/v1/phr/login/mobileEmail/init", response, 1)
+}
+async function interceptPreVerification(strBody) {
+    var response = {
+        method: 'POST',
+        body: strBody,
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    }
+    await intercept(process.env.bahmniHost + "/hiprovider/v1/phr/login/mobileEmail/preVerification", response, 1)
+}
+
+async function interceptAuthMethods() {
+    var strAuthBody = { "healthIdNumber": gauge.dataStore.scenarioStore.get("abhaNumber"), "authMethods": ["AADHAAR_OTP", "MOBILE_OTP"], "status": "ACTIVE", "blockedAuthMethods": [] }
+    var strAuthMethodBody = JSON.stringify(strAuthBody);
+    var response = {
+        method: 'POST',
+        status: 202,
+        body: strAuthBody,
+        headers: {
+            'Content-Type': 'application/json',
+            'content-length': strAuthMethodBody.length
+        }
+    }
+    await intercept(process.env.bahmniHost + "/hiprovider/v1/phr/registration/hid/search/auth-methods", response, 1)
+}
+
+async function interceptInitTransaction() {
+    var response = {
+        method: 'POST',
+        status: 202
+    }
+    await intercept(process.env.bahmniHost + "/hiprovider/v1/phr/login/init/transaction?authMode=AADHAAR_OTP", response, 1)
+}
 
+async function interceptGetUserToken() {
+    var response = {
+        method: 'POST',
+        status: 202,
+        body: {},
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    }
+    await intercept(process.env.bahmniHost + "/hiprovider/v1/phr/login/mobileEmail/getUserToken", response, 1)
+}
+
+async function interceptHID() {
+    var response = {
+        method: 'POST',
+        status: 202,
+        body: { "success": "true" },
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    }
+    await intercept(process.env.bahmniHost + "/hiprovider/v1/phr/profile/link/hid", response, 1)
+}
+
+async function interceptAadhaarVerifyOtpMatchingRecord() {
+    var verifyOtp = await replacePopulateAadharDetails(fileExtension.parseContent("./data/confirm/aadhaarDetailsForMatchingRecord.txt"));
+    var response = {
+        method: 'POST',
+        body: verifyOtp,
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    }
+    await intercept(process.env.bahmniHost + "/hiprovider/v2/registration/aadhaar/verifyOTP", response, 1)
+    gauge.message("intercepted" + process.env.bahmniHost + "hiprovider/v2/registration/aadhaar/verifyOTP")
+}
+
+async function interceptAadhaarVerifyOtpExistingABHANoABHAAddress(){
```

Comment:
This method is same as "interceptAadhaarVerifyOtpMatchingRecord" only the text file is changing.. Reuse the same method by passing file location as argument.